tests/feedback.test.ts

```diff
@@ -0,0 +1,147 @@
+import { validate } from 'class-validator';
+import { plainToClass } from 'class-transformer';
+import { DatabaseConnection, UserFactory } from './data';
+import { FeedbackFactory } from './data/FeedbackFactory';
+import { FeedbackControllerMock } from './controllers';
+import { ActivityScope, ActivityType, FeedbackStatus, UserAccessType } from '../types';
+import { ActivityModel } from '../models/ActivityModel';
+import { FeedbackModel } from '../models/FeedbackModel';
+import { Feedback } from '../api/validators/FeedbackControllerRequests';
+import { UserModel } from '../models/UserModel';
+import { Config } from '../config';
+
+beforeAll(async () => {
+  await DatabaseConnection.connect();
+});
+
+beforeEach(async () => {
+  await DatabaseConnection.clear();
+});
+
+afterAll(async () => {
+  await DatabaseConnection.clear();
+  await DatabaseConnection.close();
+});
+
+describe('feedback submission', () => {
+  test('returns proper feedback response on successful submission', async () => {
```

Comment:
Wouldn't that be an extra unnecessary query though? Also, this test is explicitly for checking the `submitFeedback` response object's validity, and if we want to check the response of `getFeedback`, then I think that should be in a different test

Reply:
I just don't get the point of the test. Like, the response object is something we return and we have a type for it so why do we need to check its validity. If we want to make sure the submission went through, then we could call GET or query the database, but I don't see the point in checking the response object since it's basically just whatever we put in the request plus some constants.